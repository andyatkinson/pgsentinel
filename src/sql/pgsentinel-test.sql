CREATE EXTENSION pg_stat_statements;
CREATE EXTENSION pgsentinel;
select pg_sleep(3);
select count(*) > 0 AS has_data from pg_active_session_history where queryid in (select queryid from pg_stat_statements);
DROP EXTENSION pgsentinel;
DROP EXTENSION pg_stat_statements;
