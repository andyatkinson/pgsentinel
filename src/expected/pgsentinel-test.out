CREATE EXTENSION pg_stat_statements;
CREATE EXTENSION pgsentinel;
select pg_sleep(3);
 pg_sleep 
----------
 
(1 row)

select count(*) > 0 AS has_data from pg_active_session_history where queryid in (select queryid from pg_stat_statements);
 has_data 
----------
 t
(1 row)

DROP EXTENSION pgsentinel;
DROP EXTENSION pg_stat_statements;
